pyproject.toml

```diff
@@ -63,7 +72,7 @@ Changelog = "https://SINTEF.github.io/ci-cd/latest/CHANGELOG"
 "ci-cd" = "ci_cd.main:program.run"
 
 [tool.mypy]
-python_version = "3.9"
+python_version = "3.7"
```

Comment:
Why do you revert from `python` 3.9 to 3.7 here?

Reply:
As written in the PR description, this is part of the update:

> Update configurations for mypy: Use Python 3.7.

Essentially, it lets mypy know that it should check typing according to what is available and the convention for Python 3.7. This makes sense, since it's the lowest supported version for the package.